ci/Jenkinsfile.linux

```diff
@@ -6,15 +6,7 @@ def isPRBuild = utils.isPRBuild()
 def isNightlyBuild = utils.isNightlyBuild()
 
 pipeline {
-  agent {
-    /* Necessary image with Ubuntu 20.04 for older Glibc. */
-    docker {
-      label 'linux'
-      image 'statusteam/nim-status-client-build:1.4.1-qt5.15.2'
-      /* allows jenkins use cat and mounts '/dev/fuse' for linuxdeployqt */
-      args '--entrypoint="" --cap-add SYS_ADMIN --security-opt apparmor:unconfined --device /dev/fuse'
-    }
-  }
+  agent { label 'linux && x86_64 && nix-2.14' }
```

Comment:
We are on `2.19` right now:
https://github.com/status-im/infra-ci/blob/67f9fc643a1e054d97c53a0e30446943d341cd98/ansible/roles/nix-setup/defaults/main.yml#L6